scapy/utils.py

```diff
@@ -470,7 +470,7 @@ def do_graph(graph,prog=None,format=None,target=None,type=None,string=None,optio
     string: if not None, simply return the graph string
     graph: GraphViz graph description
     format: output type (svg, ps, gif, jpg, etc.), passed to dot's "-T" option
-    target: filename or redirect. Defaults pipe to Imagemagick's display program
+    target: filename (if so, use -ofilename) or redirect. Defaults pipe to Imagemagick's display program
```

Comment:
Unless I'm wrong, this is no longer needed. Could you remove this?

Reply:
I don't get it. How could it be no longer needed ?
Latest GraphViz's version requires that one use `-ofilename` with no space. As this is a quite uncommon format, i thought it had to be specified in the doc :/
What I can do is unify the -o params so it does not need to be specified through the args.

Reply:
OK I had missed the Windows part of the story. I though it was OK under a windows shell to redirect stdout. It does not work with GraphViz?

Reply:
My Bad, I'm wrong.
`>` works on Windows (only using subprocess on shell), but as the original doc said `target` could be a filename, which did not work, i thought there was a bug...

I fixed it by adding a safety check about the type of target